### PR TITLE
feat: t2005 files added

### PR DESCRIPTION
**Pull request by Abishek666**
In this PR, I added the class for creating object for different cities for easy use by converting function prototype into class. And i have added few unit testing code for different functions

### CODE REVIEW

**Comment by saikrishnanp** [comment]
Let's start adding description from next PRs 
image
It can have what's the PR about and how was the implementation done and if has any testing added together

**Comment by saikrishnanp** [comment]
I found a behavior where we are able to scroll the page but, when hovering over the middle section and scroll it doesn't scroll the page. Can we fix this?
weather app scroll
You can feel free to add fix for this in same PR or a follow up PR. Both are fine

File: src/index.js [hunk 1]
```diff
@@ -2,18 +2,18 @@ import { domElement } from './domElements.js'
 import { topSection } from './topSection.js'
 import { bottomSection } from './bottomSection.js'
 import { middleSection } from './middleSection.js'
-import * as topProto from './basePrototype.js'
-// import * as topClass from './baseClass.js'
+// import * as topProto from './basePrototype.js'
```
Comment (saikrishnanp) [inline src/index.js:5]: Let's remove the unneeded commented lines

File: src/index.js [hunk 2]
```diff
@@ -2,18 +2,18 @@ import { domElement } from './domElements.js'
...
+import * as topClass from './baseClass.js'
```
Comment (saikrishnanp) [inline src/index.js:6]: If a single class or method is needed then we need not import everything(*)

**Review by saikrishnanp — APPROVED** [review]

**Review by saikrishnanp — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (Abishek666) [reply src/index.js:5]: sure I will resolve it in next pr

File: src/index.js (hunk 2 above)
Comment (Abishek666) [reply src/index.js:6]: sure I will resolve it in next pr

**Pull request merged**